public/fonts

```diff
@@ -0,0 +1 @@
+../node_modules/@toniq-labs/design-system/public/fonts
```

Comment:
These font files are not getting loaded in the Netlify build. At build time, replace these sym links with the actual files, or figure out how to get webpack to respect symlinks.